nltk/decorators.py

```diff
@@ -79,6 +95,7 @@ def getinfo(func):
         name=func.__name__,
         argnames=argnames,
         signature=signature,
+        fullsignature=fullsignature,
```

Comment:
I kept signature untouched, and added a `fullsignature` for the standard Signature object.

A cleaner version would be to replace `signature` with the Signature object. But it may break some user's code.
I'm open to suggestions if you have a better solution.